sdk/java/src/main/java/com/gojek/feast/FeastClient.java

```diff
@@ -161,9 +162,9 @@ public GetFeastServingInfoResponse getFeastServingInfo() {
         .collect(Collectors.toList());
   }
 
-  private FeastClient(ManagedChannel channel) {
+  protected FeastClient(ManagedChannel channel) {
```

Comment:
same here, leave it as private

Reply:
^ Refer to above.

Reply:
I only need to mock `channel`. Mocking `stub` is not possible as its a `final` class.

Reply:
okayy